jck/runtime.lang/playlist.xml

```diff
@@ -265,7 +226,7 @@
 			<group>jck</group>
 		</groups>
 		<subsets>
-			<subset>11+</subset>
+			<subset>9+</subset>
```

Comment:
According to the comment in 352, jck-runtime-lang-MOD is only defined in 11+

Reply:
Similar to above, MOD is related to modularity, it was introduced from JCK 9. It was marked as 11+ in the playlist as we had discontinued testing JCK 9, but to be precise, it should be 9+, hence the change.